xml2rfc_compat/serializer.py

```diff
@@ -18,4 +18,22 @@ def to_xml_string(item: BibliographicItem, **kwargs) -> bytes:
     Delegates to ``relaton-py`` implementation
     of :mod:`relaton.serializers.bibxml_string` serializer.
     """
-    return serialize(item, **kwargs)
+    # get a tree
+    canonicalized_tree = etree.fromstring(
+        # obtained from a canonicalized string representation
+        etree.tostring(
+            # of the original bibxml tree
+            _original_serialize(item, **kwargs),
+            method='c14n2',
+        )
+        # ^ this returns a unicode string
+    )
+
+    # pretty-print that tree in utf-8 with declaration and doctype
```

Comment:
@stefanomunarini yeah, note that the `declaration` and `doctype` mention doesn’t apply anymore in this comment